docs-src/concepts.md

```diff
@@ -128,19 +125,10 @@ A typical client/gateway API request flow for a gateway implemented using a reve
 
 ## TLS Configuration
 
-TLS configuration is tied to Gateway listeners. Although adding the option to
-configure TLS on other resources was considered, ultimately TLS configuration on
-Gateway listeners was deemed sufficient for the following reasons:
-
-* In most cases, users that are configuring TLS will naturally also have access
-  to Gateways.
-* In other cases, TLS config could be implemented with a controller watching
-  Routes and adding generated certs to corresponding Gateways.
-* This does not solve the use case for users wanting to provide their own certs
-  for Routes while not having access to a Gateway resource. This seems like it
-  would be a rare edge case and is not worth supporting at this point. The
-  security model outlined a potential approach to enable this in the future, but
-  there does not seem to be a sufficient reason to work towards that now.
+TLS configuration is tied to Gateway listeners. Additionally, TLS certificates
```

Comment:
COnsider the following minor change:

s/TLS configuration is tied to Gateway listeners./TLS is configured on Gateway listeners./